src/index.scss

```diff
@@ -3,7 +3,3 @@
 
 $fa-font-path: "~font-awesome/fonts";
 @import "~font-awesome/scss/font-awesome";
-
-button:hover {
```

Comment:
Good catch on the version of `edx-bootstrap` so that we can remove this.

Reply:
Followup PR to frontend-cookiecutter coming